src/main/java/gov/cms/madie/models/measure/QdmMeasure.java

```diff
@@ -17,7 +17,17 @@
 @ToString(callSuper=true)
 public class QdmMeasure extends Measure {
 
-	@NotBlank(message = "Scoring is required for QDM Measure.")
+	@EnumValidator(
+      enumClass = MeasureScoring.class,
+      message = "Scoring must be a valid MADiE scoring type"
+  )
 	private String scoring;
 	
+	@Override
+  public void setGroups(List<Group> groups) {
+    if (groups.stream().map(Group::getScoring).allMatch(scoring -> groups.get(0).getScoring().equalsIgnoreCase(scoring))
+    		&& groups.get(0).getScoring().equalsIgnoreCase(this.scoring)) {
+      super.setGroups(groups);
+    } else throw new RuntimeException("Groups must have same scoring");
```

Comment:
Does this mean, In UI we are not going to have a base configuration?

Reply:
ooh my bad, scoring is still on Measure level but is also on all groups level. User has to go through 2 step process in order to set scoring for all groups. 

I don't understand the requirement I guess.

Reply:
@RohitKandimalla We can just automatically set group scoring when groups are created/updated and make the field read only if shown.

Reply:
So if we are setting the scoring on measureGroups, why do we need a check in the backend?

Reply:
@RohitKandimalla The UI part for now is on Base Configuration, which sets the scoring on the Measure level. When it comes to Population Criteria later, we can use the existing Group model, and set the scoring at the backend.

Reply:
Ok I am not understanding the requirements for this PR accurately. so I am going to resolve this

Reply:
I was thinking when the user sets the scoring on the Base Configuration UI, the backend will create a nearly empty group to hold that scoring value. The Group model already has most of the needed validations expect for enforcing all Groups have the same scoring.

Instead of having a 1-to-1 matching of UI and model (ie. adding QdmMeasure.scoring), we store the Scoring value in a Group and then add a constraint at the QdmMeasure level that makes sure any new groups have the same scoring. 

The Base Configuration UI will have to pull the Scoring value from the first Group. Also, the UI for QDM Measures will never display the Scoring value in the Pop Criteria pages.